### ISSUE TITLE
Reading big DLIS files: MemoryError: Unable to allocate array with shape

### ISSUE THREAD

**Issue by cimka** [equinor/dlisio#247]
While reading a big DLIS file we get MemoryError: Unable to allocate array with shape....

After discussing with @ErlendHaa 
I think we would like to be able to send in a list of mnemonics we would like to extract from the dlis file.
Something like this:
1. Read available mnemonics.
2. Extract data from mnemonics we supply.

`alloc = lambda size: np.empty(shape = size, dtype = dtype)
MemoryError: Unable to allocate array with shape (5473,) and data type......`

**Comment by jokva** [comment]
Are you running it on Windows? 64-bit python? How large is this file, really?

**Comment by cimka** [comment]
I run on Windows 64 bit version and the file in question is 1GB.
This above is one example, had more and saw memory consumption of python was at times above 25GB (Machine has 32GB).

**Comment by jokva** [comment]
That sounds more like a bug, rather than the approach being wrong.

**Comment by cimka** [comment]
The full list of mnemonics that is trying to be allocated (dont wanna post them all here) indicate that allot of memory wants to be assigned.
Can do more distinct testing just to verify later on.

**Comment by achaikou** [comment]
I saw this issue before, on another file which had one Frame of over 1 GB size.

I opened file we are talking about from my Linux machine, but I was unable to reproduce the error, frame.curves work just fine.
I don't remember what I did before, but the problem was gone. I think it was either switching to newer release, or somehow blaming network connection and downloading file locally. But nothing helped me to reproduce the error now, so I must be wrong.
